blog/_src/posts/2016-05-18-gradual-typing-across-the-spectrum.md

```diff
@@ -0,0 +1,181 @@
+    Title: Gradual Typing Across the Spectrum
+    Date: 2016-05-18T07:58:56
+    Tags: gradual typing, PI meeting, by Asumu Takikawa
+
+> Instead of being Pythonistas, Rubyists, or Racketeers we have to
+> be scientists. -- Matthias Felleisen
+
+Yesterday we hosted a PI meeting for the Gradual Typing Across the Spectrum
+NSF grant, gathering researchers from a number of institutions who work on
+gradual typing. In case you aren't familiar with gradual typing, the idea is
+to augment dynamically typed languages (think Python or Ruby) with static
+type annotations (as documentation, for debugging, or for tool support)
+that are guaranteed to be sound.
+
+Gradual typing is these days a fairly popular area, but
+the research can seem fragmentary
+because of the need to support idiosyncratic language features. One of the
+points of the meeting was to encourage the cross-pollination of the key
+scientific ideas of gradual typing---the ideas that cross language
+and platform barriers.
+
+<!-- more -->
+
+There were a good number of both institutions and programming languages
+represented at the meeting, with researchers from all of [Brown University](http://cs.brown.edu/research/plt/),
+[Indiana University](https://wonks.github.io/), [Northeastern University](http://prl.ccs.neu.edu/),
+and the [University of Maryland](http://www.cs.umd.edu/projects/PL/).
+The languages that we work on cover a broad subset of the dynamically-typed
+languages: Clojure, JavaScript, R, Racket, Ruby, Pyret, and Python.
+
+The specific research artifacts that were represented include
+[Reticulated Python](https://github.com/mvitousek/reticulated),
+[RDL](https://github.com/plum-umd/rdl),
```

Comment:
RDL is the only name in this list I couldn't make sense of. Maybe `RDL (contracts for Ruby)` would clarify for the reader?